sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/NonAsyncTaskShouldNotReturnNull.cs

```diff
@@ -78,8 +78,8 @@ private static SyntaxNode GetEnclosingMember(LiteralExpressionSyntax literal)
                 switch (ancestor.Kind())
                 {
                     case SyntaxKind.ParenthesizedLambdaExpression:
+                    case SyntaxKind.SimpleLambdaExpression:
```

Comment:
I thought the same way, but wasn't sure either whether that was the correct way.
Since we both agree, let's got with it, and see the result.
To feel condifent about this change, as discussed off-line, let's delay merging this PR, so as to not include it in next release, in order to properly test and check issue results on larger code base